app/assets/javascripts/modules/protected_areas/base.js.coffee

```diff
@@ -4,7 +4,7 @@ $(document).ready( ->
   )
 
   require(['show_hide'], (ShowHide) ->
-    new ShowHide($('.js-show-hide'))
+    new ShowHide($('.js-show-hide'), 6)
```

Comment:
It's a good idea to pass this value as a parameter as well. ;)
What I also mean is not to pass/use values as plain integer, but it might be a good idea to define a constant variable that contains that number.
It encourages code recycling and readability, so everybody knows exactly what that number represents :)